{{cookiecutter.project_slug}}/README.rst

```diff
@@ -0,0 +1,3 @@
+{% for x in cookiecutter.project_name|list() %}={% endfor %}
+{{ cookiecutter.project_name }}
+{% for x in cookiecutter.project_name|list() %}={% endfor %}
```

Comment:
I've taken inspiration from the pydanny cookiecutter-django and changed the header a bit - should hopefully be a bit more obvious as to what it's generating.